lib/http-server.js

```diff
@@ -58,13 +58,13 @@ function HttpServer(options) {
 
   var before = options.before ? options.before.slice() : [];
 
-  before.push(function (req, res) {
-    if (options.logFn) {
+  if (options.logFn) {
+    before.push(function (req, res) {
       options.logFn(req, res);
-    }
+    })
 
     res.emit('next');
```

Comment:
This needs to be inside the `before.push(function (req, res) { })` otherwise it will be undefined